### ISSUE TITLE
polygons_full is not callable (with Python 3.8.2 environment)

### ISSUE THREAD

**Issue by HadrienLMS** [mikedh/trimesh#819]
Hello!

Congrats on the nice Library. I am cutting CAD objects by hyperplanes and it seems trimesh is the best option.

Reporting on a nasty issue though (as far as I can tell because I am new to Python), which is environment dependent. I am using a code template from there https://trimsh.org/examples/section.html, so it should (in principle) not be related to a coding mistake. 

The attribute polygons_full is somehow not callable when I use a Python 3.8.2 environment. Everything seems to work fine apart from that, polygons_closed for instance is callable! When I use a Python 3.7 environment it works like a charm. 

I have enclosed the two conda environments to reproduce the error. 

It might be that someone else alluded to this back in 2018 (see https://github.com/mikedh/trimesh/issues/210)  

"I looked into this a bit - and it looks to me that this is because polygons in polygons_full are sometimes instances of MultiPolygon instead of Polygon?"

to which you replied 

"No worries! Hmm, polygons_full should definitely be a regular Polygon object. Would you be able to mail me the model and script that produced this traceback? Thanks! Mike"

But then it seems a workaround was found and issue (probably) not addressed. I attach here the description of the environments and the python script.
Many thanks for your help,

Hadrien

[IssueReport.zip](https://github.com/mikedh/trimesh/files/4585482/IssueReport.zip)

![Issue](https://user-images.githubusercontent.com/7046306/81149228-176ea580-8f7e-11ea-92e3-86ee44f17db0.png)

**Comment by mikedh** [comment]
I think you're missing `rtree`? The error message is unhelpful which I'll fix. Does `from rtree import Rtree` work?

**Comment by HadrienLMS** [comment]
Thank you for the quick reply. In the meantime I reverted to the initial Anaconda distribution, did a few trial and errors and now everything works fine. 

A few conclusions from my somewhat tedious explorations (w/ Windows 10 + Anaconda3) is that 

 pip install trimesh[easy] was not satisfactory - I got everything installed but got (yet another) issue "OSError: [WinError 126] The specified module could not be found" when executing from .exchange.load import load_path in the m.section method. 

So I tried "conda install -c conda-forge trimesh" instead but then noticed that it does not come with the soft dependencies (shapely etc) when I thought it was indicated otherwise in the installation guide of trimesh. 

So I installed manually shapely, and then triangle using pip (could not find a conda version) as it complained it was missing in the remeshing of the cap. 

All in all what I did is (right after conda refreshing to revision v1) 

conda install pip
conda install -c conda-forge trimesh
conda install -c conda-forge shapely
pip install triangle

and it worked fine for STL mesh slicing and cap re-meshing
Hope that helps,

Hadrien
